server/watchers.js

```diff
@@ -90,13 +90,13 @@ export function getPayPalSDKWatcher({ logBuffer, cache } : {| logBuffer : ?Logge
     };
 }
 
-export function getPayPalSmartPaymentButtonsWatcher({ logBuffer, cache } : {| logBuffer : ?LoggerBufferType, cache : ?CacheType |}) : Watcher {
+export function getPayPalSmartPaymentButtonsWatcher({ logBuffer, cache, locationInformation } : {| logBuffer : ?LoggerBufferType, cache : ?CacheType, locationInformation : InstanceLocationInformation |}) : Watcher {
     if (!cache || !logBuffer) {
         throw new Error(`Cache and logBuffer required`);
     }
 
     paypalSmartButtonsWatcher = paypalSmartButtonsWatcher || poll({
-        cdnRegistry:  SMART_BUTTONS_CDN_NAMESPACE,
+        cdnRegistry:  `https://${ locationInformation.cdnHostName || CDN_NAMESPACE }/${ SMART_BUTTONS_CDN_NAMESPACE }`,
```

Comment:
`cdnHostName` will always have a value. No need to fallback to a hardcoded domain

```suggestion
        cdnRegistry:  `https://${ locationInformation.cdnHostName }/${ SMART_BUTTONS_CDN_NAMESPACE }`,
```